src/scripts/recon/allowed-methods/__main__.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+from requests import request
+from requests.exceptions import *
```

Comment:
Naaaah bro, it's better not to use wildcard imports, be careful!
https://docs.quantifiedcode.com/python-anti-patterns/maintainability/from_module_import_all_used.html

If you need something, just took something directly :)